### ISSUE TITLE
Sandbox issue - access home directory

### ISSUE THREAD

**Issue by egrath** [flathub/org.gnome.Genius#2]
The Flatpack is not allowed to access the home directory of the executing user, so it's not possible to:

1. Save Genius configuration under ~/.gnome2/genius (as the sandbox gets cleaned after termination)
2. Load/Save Files to the home directory.

**Comment by TingPing** [comment]
> Save Genius configuration under ~/.gnome2/genius (as the sandbox gets cleaned after termination)

This should be patched in Genius, that is a terrible directory to use.

In the short term adding the permission `--persist=.gnome2` will work.

> Load/Save Files to the home directory.

It should be using GtkFileChooserNative and that fixes that.

**Comment by egrath** [comment]
Just if someone else stumbles across this issue. The command to issue is:
`flatpak --user override --filesystem=home org.gnome.Genius` or alternatively, if you only want to allow to save the configuration `flatpak --user override --persist=.gnome2 org.gnome.Genius`

**Comment by nedrichards** [comment]
This is now not a dead GTK2 application, a new release has appeared with a GTK3 port. It still doesn't use the portal enabled filechooser though...

**Comment by TingPing** [comment]
The question is just what config dir does it use now?

**Comment by wjt** [comment]
I folded @egrath's fix for the config dir (#6) into my PR to update the app, its dependencies and its runtime (#7). I confirmed that that version does use the file chooser portal.

```
wjt@camille:~$ flatpak permission-show org.gnome.Genius | grep documents
documents	f767535	org.gnome.Genius	read,write,grant-permissions	(b'/sysroot/home/wjt/Documents/Program_1.gel', 66307, 1827977, 0)
```
